app/controllers/oidc_controller.rb

```diff
@@ -111,7 +113,7 @@ def client
       authorization_endpoint: config.authorization_endpoint,
       token_endpoint:         config.token_endpoint,
       userinfo_endpoint:      config.userinfo_endpoint,
-      redirect_uri:           url_for(controller: "oidc", action: "done")
+      redirect_uri:           dashboard + oidc_path
```

Comment:
I'd better use: `url_for(controller: "oidc", action: "done", host: Pillar.value(pillar: :dashboard_external_fqdn))`. Let's use the helper with the `host` parameter, and also let's use the external fqdn for the dashboard, since this is where dex will redirect the user, and this host needs to be resolvable by the user.

Reply:
Or even better, we can use the implicit url helper: `oidc_done(host: Pillar.value(pillar: :dashboard_external_fqdn))`

Reply:
updated, thanks